windows/benji.py

```diff
@@ -417,6 +418,67 @@ def main():
 			speak.say("Sorry Graph can not be Plotted")
 			speak.runAndWait()
 
+	#Box Office Status
+	elif link[-1] == "boxoffice":
+		try:
+			url = "https://in.bookmyshow.com/" + link[0] + "/movies/nowshowing"
+			r =  requests.get(url)
+			soup = BeautifulSoup(r.content, 'html.parser')
+
+			soup_level2 = []
+			show_status_list = []
+			shows_list = soup.find_all('div', attrs={'class': 'card-container wow fadeIn movie-card-container'})
+			for i in shows_list:
+				start = str(i).index("href=")
+				end = str(i).index("title=")
+				soup_level2.append("https://in.bookmyshow.com" + str(i)[start+6 : end-2])
+
+			show_status_raw = soup.find_all('div', attrs={'class': 'popularity sa-data-plugin'})
+			for i in show_status_raw:
+				start = str(i).index("data-coming-soon=")
+				end = str(i).index('data-event-code')
+				data = str(i)[start+18 : end-2]
+				
+				if data == "false":
+					show_status_list.append("In Cinemas Now...")
+				if data == "true":
+					show_status_list.append("Coming Soon...")
+
+			Tags_list = []

```

Comment:
Instead of using 3 lists tags list, name list and show status list, you can create a dictionary of items with the name being the key and the tags being the value, and get rid of the status list altogether
Something like:
```
{"Now showing": {name: taglist}, {name2:taglist},
"Coming soon": {name3:taglist},{name4:taglist}
}
```
Also, the tag list for each name should be a `set`, not a list, t avoid duplicate tags.
Not that lists are wrong, but a `dict` will be a better presentation of this data. WDYT?

Reply:
I can't do it , though status and names are scraped with different levels of the soup.